packages/expo-modules-core/ios/Interfaces/FileSystem/EXFileSystemInterface.h

```diff
@@ -17,7 +17,7 @@ typedef NS_OPTIONS(unsigned int, EXFileSystemPermissionFlags) {
 
 // TODO: Move permissionsForURI to EXFileSystemManagerInterface
 - (EXFileSystemPermissionFlags)permissionsForURI:(NSURL *)uri;
-- (NSString *)generatePathInDirectory:(NSString *)directory withExtension:(NSString *)extension;
+- (nonnull NSString *)generatePathInDirectory:(NSString *)directory withExtension:(NSString *)extension;
```

Comment:
I've marked it that way as it correctly resembles the logic and Swift was complaining about nullability of the return value.